docs(bmaas): add shared responsibility model (#3486)
scaleway/docs-content: bare-metal/elastic-metal/reference-content/shared-responsibility-model.mdx

---
meta:

>   title: Scaleway Bare Metal Shared Responsibility Mosdel
>   description: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
> content:
>   h1: Scaleway Bare Metal Shared Responsibility Mosdel

Reviewer: ```suggestion
  title: Scaleway Bare Metal Shared Responsibility Model
  description: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
content:
  h1: Scaleway Bare Metal Shared Responsibility Model
```

  paragraph: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
tags: bare metal shared responsibility
dates:
  validation: 2024-07-18
  posted: 2024-07-18
categories:
  - bare-metal
---

On this page, we outline the roles and responsibilities for maintaining and securing your Bare Metal servers.
Our shared responsibilty model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 
By understanding this shared responsibility, you can optimize the performance, reliability, and security of your Scaleway Bare Metal services.

## Product resiliency

### Availability

> Bare Metal Products targets the following power supplied and connectivity SLOs:

Reviewer: ```suggestion
Bare Metal Products target the following power supplies and connectivity SLOs:
```

The detailed SLAs measurements and guarantees can be found at [Service Level Agreement for Bare Metal Services](https://www.scaleway.com/en/bare-metal/sla/).

Once the resource is in your hands, we have no access to the machine and therefore no way to monitor its operation.
In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resource as quickly as possible.

### Backups and snapshots

You are fully responsible for managing your machine and its data.

It is your responsibility to ensure data redundancy if necessary, by relying on backups or internal server redundancy through solutions such as RAID.

In the event of a disk or a hardware failure, we cannot guarantee that you will regain access to your machine and data. The failure may require a complete replacement of the machine. Therefore, it is your responsibility to ensure in advance that you have a backup or redundancy of your data on another server. We cannot be held responsible for the loss of your data.

> Backups and snapshots must be configured by you. Please refer to our documentation for assistance in setting up backups for [Dedibox](https://www.scaleway.com/en/docs/dedibox/dedicated-servers/how-to/use-dedibackup-ftp-backup/), [Elastic Metal](https://www.scaleway.com/en/docs/faq/elastic-metal/#how-do-i-back-up-an-elastic-metal-server), and [Apple silicon](https://www.scaleway.com/en/docs/faq/apple-silicon/#how-can-i-back-up-my-mac-mini).

Reviewer: ```suggestion
Backups and snapshots must be configured by you. Please refer to our documentation for assistance in setting up backups for [Dedibox](/dedibox/dedicated-servers/how-to/use-dedibackup-ftp-backup/), [Elastic Metal](/faq/elastic-metal/#how-do-i-back-up-an-elastic-metal-server), and [Apple silicon](/faq/apple-silicon/#how-can-i-back-up-my-mac-mini).
```

</Message>

## Data protection

### Encryption in transit

Bare Metal servers supports SSH connections to secure your communication with the server. You remain responsible for configuring SSH keys. 

### Data encryption

You are responsible for encrypting the disks on your server. We are not responsible for data encryption, especially in cases of sensitive applications or additional security requirements.

### Data deletion

When you delete your server, we are responsible for removing all of your data. To ensure their protection, the deleted server remains attached to your account and is therefore inaccessible to anyone else until all of your data has been erased and the cleaning process has been completed with a 100% success rate.

In the event of a defective storage disk, it will be destroyed to ensure the confidentiality of your data.

### Scaleway access

We do not have the technical capability to access your machine once it is installed, nor the data stored on it. We have no visibility into your use of the server and its configuration. Therefore, it is your responsibility to ensure the security of your machine and data.

## Identity and access management

> Elastic Metal servers provide [IAM permissions](https://www.scaleway.com/en/docs/identity-and-access-management/iam/reference-content/permission-sets/) sets which allow or restrict specific actions a user or application can perform, such as creating or deleting Elastic Metal servers. You remain responsible for giving these permissions to the relevant users or applications and reviewing these accesses frequently.

Reviewer: ```suggestion
Elastic Metal servers provide [IAM permissions](/identity-and-access-management/iam/reference-content/permission-sets/) sets which allow or restrict specific actions a user or application can perform, such as creating or deleting Elastic Metal servers. You remain responsible for giving these permissions to the relevant users or applications and reviewing these accesses frequently.
```

---
meta:
  title: Scaleway Bare Metal Shared Responsibility Model
  description: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
content:
  h1: Scaleway Bare Metal Shared Responsibility Model
  paragraph: Learn about the shared responsibility model for Scaleway Bare Metal services, outlining the roles of Scaleway and users in managing server security, backups, and compliance.
tags: bare metal shared responsibility
dates:
  validation: 2024-07-18
  posted: 2024-07-18
categories:
  - bare-metal
---

On this page, we outline the roles and responsibilities for maintaining and securing your Bare Metal servers.

> Our shared responsibilty model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 

Reviewer 2: ```suggestion
Our shared responsibility model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 
```

> Our shared responsibilty model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 

Reviewer 3: ```suggestion
Our shared responsibility model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing server availability, backups, configurations, and security measures. 
```

By understanding this shared responsibility, you can optimize the performance, reliability, and security of your Scaleway Bare Metal services.

## Product resiliency

### Availability

Bare Metal Products target the following power supplies and connectivity SLOs:

* For Dedibox servers:
    * Start: 99.9% availability
    * Pro: 99.95% availability
    * Core and Store: 99.99% availability
* For Elastic Metal servers:
    * Aluminium: 99.9% availability
    * Beryllium and Lithium: 99.95% availability
    * Iridium: 99.99% availability
* For Apple silicon servers:
    * Mac mini: 99.95% availability

The detailed SLAs measurements and guarantees can be found at [Service Level Agreement for Bare Metal Services](https://www.scaleway.com/en/bare-metal/sla/).

Once the resource is in your hands, we have no access to the machine and therefore no way to monitor its operation.

> In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resource as quickly as possible.

Reviewer 2: ```suggestion
In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resources as quickly as possible.
```

### Backups and snapshots

You are fully responsible for managing your machine and its data.

It is your responsibility to ensure data redundancy if necessary, by relying on backups or internal server redundancy through solutions such as RAID.

In the event of a disk or a hardware failure, we cannot guarantee that you will regain access to your machine and data. The failure may require a complete replacement of the machine. Therefore, it is your responsibility to ensure in advance that you have a backup or redundancy of your data on another server. 
**We cannot be held responsible for the loss of your data.**

Backups and snapshots must be configured by you. Please refer to our documentation for assistance in setting up backups for [Dedibox](/dedibox/dedicated-servers/how-to/use-dedibackup-ftp-backup/), [Elastic Metal](/faq/elastic-metal/#how-do-i-back-up-an-elastic-metal-server), and [Apple silicon](/faq/apple-silicon/#how-can-i-back-up-my-mac-mini).
Restoring snapshots or backups needs to be triggered by you.

## Configuration and version management

### Installation and configuration

> We provide a range of Linux, Windows ans macOS server distributions for automatic installation from the Scaleway console. These distributions come with a default configuration designed for standard use cases, ensuring security, usage efficiency, and reliability. During the setup of your machine, you can modify and customize this initial configuration. However, you are responsible for any impact on your server's availability, security, or performance.

Reviewer 2: ```suggestion
We provide a range of Linux, Windows and macOS server distributions for automatic installation from the Scaleway console. These distributions come with a default configuration designed for standard use cases, ensuring security, usage efficiency, and reliability. During the setup of your machine, you can modify and customize this initial configuration. However, you are responsible for any impact on your server's availability, security, or performance.
```

> We provide a range of Linux, Windows ans macOS server distributions for automatic installation from the Scaleway console. These distributions come with a default configuration designed for standard use cases, ensuring security, usage efficiency, and reliability. During the setup of your machine, you can modify and customize this initial configuration. However, you are responsible for any impact on your server's availability, security, or performance.

Reviewer 3: ```suggestion
We provide a range of Linux, Windows, and macOS server distributions for automatic installation from the Scaleway console. These distributions come with a default configuration designed for standard use cases, ensuring security, usage efficiency, and reliability. During the setup of your machine, you can modify and customize this initial configuration. However, you are responsible for any impact on your server's availability, security, or performance.
```

> Apple Silicon servers, in particular, are enrolled with a Scaleway operated mobile device management server. Removing the enrolment, or any of the associated configuration profiles, may impact Scaleway's ability to monitor & restart your server on your behalf, for instance, in case of power loss.

Reviewer 3: ```suggestion
Apple Silicon servers, in particular, are enrolled with a Scaleway-operated mobile device management server. Removing the enrolment, or any of the associated configuration profiles, may impact Scaleway's ability to monitor & restart your server on your behalf, for instance, in case of power loss.
```

In the event that you use a custom image, it is your responsibility to ensure a reliable and secure configuration of your machine.

### Updates and version management

We regularly provide OS version updates, allowing you to upgrade your environment if desired. It is your responsibility to update your machine to the desired version and thus maintain its compatibility with all internal and external resources at Scaleway.

If you perform manual upgrades without reinstalling your machine with an image provided by Scaleway, it is your responsibility to ensure the reliability and stability of your machine's configuration.

### Usage compliance

> You are responsible for the proper use of your resource. In this regard, you are responsible for maintaining the use of your server in accordance with Scaleway's compliance policy, as well as those of the various operating systems you use.

Reviewer 2: ```suggestion
You are responsible for the proper use of your resources. In this regard, you are responsible for maintaining the use of your server in accordance with Scaleway's compliance policy, as well as those of the various operating systems you use.
```

<Message type="important">
    It is your responsibility to inform yourself beforehand about the permitted and prohibited use cases for your server and to adhere to them throughout your usage.
</Message>

## Data protection

### Encryption in transit

> Bare Metal servers supports SSH connections to secure your communication with the server. You remain responsible for configuring SSH keys. 

Reviewer 2: ```suggestion
Bare Metal servers support SSH connections to secure your communication with the server. You remain responsible for configuring SSH keys. 
```
